pkg/minikube/cruntime/cruntime.go

```diff
@@ -178,13 +179,22 @@ func disableOthers(me Manager, cr CommandRunner) error {
 		if r.Name() == me.Name() {
 			continue
 		}
+
+		// Don't disable containerd if we are bound to it
+		if me.Name() == "Docker" && r.Name() == "containerd" && dockerBoundToContainerd(cr) {
```

Comment:
is "Docker" capitalized ?

Reply:
Yes. It's the string we output to console :(